examples/llm/src/models/layers/gpt_blocks.py

```diff
@@ -36,9 +38,13 @@ def __init__(self,
         super().__init__()
         if cfg.get('alibi', False):
             assert cfg.attn_impl == 'triton' or cfg.attn_impl == 'torch', 'Only triton kernel or torch supports alibi'
-        self.ln_1 = nn.LayerNorm(cfg.d_model, device=device)
+
+        layernorm_class = nn.LayerNorm if not cfg.get(
+            'low_precision_layernorm', False) else LPLayerNorm
```

Comment:
```suggestion
        layernorm_class = LPLayerNorm if cfg.get(
            'low_precision_layernorm', False) else nn.LayerNorm
```

nit: avoids the `not`